samza-log4j/src/main/java/org/apache/samza/logging/log4j/StreamAppender.java

```diff
@@ -279,8 +285,17 @@ protected Config getConfig() {
     return config;
   }
 
+  private void addDropProducerErrorsConfig() {
+    Map<String, String> newConfig = new HashMap<>();
+    if (config.get(DROP_PRODUCER_ERRORS) == null) {
```

Comment:
What do you think about using `config.containsKey()` here?

Reply:
Makes sense. Done.